src/app/_constants/paths.ts

```diff
@@ -43,19 +44,23 @@ function createPathObject({
   includesEntries = false,
   customMainContentPath,
   customEntriesContentPath,
+  hasSubpaths = false,
 }: PathObjectParams): PathConfig {
```

Comment:
The object parameter for this function is getting big and a bit confusing. 

The first two parameters are required and related to routing, the rest are optional and related to the CMS configuration. So perhaps we can rework the parameters as below?

```ts
function createPathObject(path, label, cmsConfig) {
  const {
    includesEntries = false,
    customMainContentPath,
    customEntriesContentPath,
    hasSubpaths = false,
  } = cmsConfig;

  // ...
}
```

I feel like `includesEntries` and `hasSubpaths` make more sense when grouped under `cmsConfig` (not attached to this name, just a suggestion)

Reply:
Yes, you are right.